src/styles/code.scss

```diff
@@ -62,6 +62,13 @@ code:not([class]) {
   margin: var(--amplify-space-xs) 0;
   padding-block: var(--amplify-space-xs);
   font-size: var(--amplify-font-sizes-medium);
+  /* 
```

Comment:
Normally I would put the un-prefixed property last (with the expectation browsers that don't yet support it would just ignore it), but when testing on iOS it wasn't picking up the vendor prefixed version and the un-prefixed version wasn't working. 🤔 Reversing them fixes it.